src/layer-manager/lib/generators/basemap/basemap.js

```diff
@@ -0,0 +1,28 @@
+import layersDirectory from './basemap-layers.json'
+
+export const BASEMAP_TYPE = 'BASEMAP'
+
+class BasemapGenerator {
+  constructor() {
+    this.type = BASEMAP_TYPE
```

Comment:
Why not:
```
class BasemapGenerator {
   type = BASEMAP_TYPE
   ...
}
```

Instead of using a constructor?

Reply:
:+1